src/Http/Controllers/OAuthController.php

```diff
@@ -124,8 +124,10 @@ public function handleProviderCallback(Request $request, string $provider)
             ]);
         }
 
+        $is_autocreate = config('socialstream.auto_create');
```

Comment:
Please use camelCase for variable names

Reply:
Changes made https://github.com/joelbutcher/socialstream/pull/49/commits/a1cea6eb550b52f3102dca5ab61b0446b675436e